FlowCrypt/src/main/java/com/flowcrypt/email/ui/activity/fragment/MessageDetailsFragment.kt

```diff
@@ -1729,6 +1763,42 @@ class MessageDetailsFragment : BaseFragment<FragmentMessageDetailsBinding>(), Pr
     }
   }
 
+  private fun bindToAttachmentDownloadManagerService() {
+    Intent(requireContext(), AttachmentDownloadManagerService::class.java).also { intent ->
+      context?.bindService(intent, downloadAttachmentsServiceConnection, Context.BIND_AUTO_CREATE)
+    }
+  }
+
+  private fun handleLoadingProgress(
+    mapWithLatestProgress: Map<String, AttachmentDownloadManagerService.DownloadProgress>
+  ) {
+    val uniqueStringIdSetToBeUpdated = mutableSetOf<String>()
```

Comment:
maybe `idsToBeUpdated ` would be better naming?
Also for function name, i think `updateProgress` would be sufficient?